platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/network/connectivity/InboundConnectionHandlerTest.java

```diff
@@ -64,17 +64,18 @@ void handleInboundOnePeerTest(final AddressBook addressBook, final Map<NodeId, K
                 .limit(2)
                 .boxed()
                 .toList();
-        final NodeId thisNode = addressBook.getNodeId(nodeIndexes.get(0));
-        final NodeId otherNode = addressBook.getNodeId(nodeIndexes.get(1));
-        final KeysAndCerts keysAndCerts1 = keysAndCerts.get(thisNode);
-        final KeysAndCerts keysAndCerts2 = keysAndCerts.get(otherNode);
-        final List<PeerInfo> peerInfoList = Utilities.createPeerInfoList(addressBook, otherNode);
-        final NetworkPeerIdentifier identifier = new NetworkPeerIdentifier(platformContext, peerInfoList);
+        final NodeId node1 = addressBook.getNodeId(nodeIndexes.get(0));
+        final NodeId node2 = addressBook.getNodeId(nodeIndexes.get(1));
+        final KeysAndCerts thisKeysAndCerts = keysAndCerts.get(node1);
+        final KeysAndCerts OtherKeysAndCerts = keysAndCerts.get(node2);
```

Comment:
variable name is incorrectly capitalized`OtherKeysAndCerts` -> `otherKeysAndCerts`